go.mod

```diff
@@ -1,9 +1,12 @@
-module howett.net/plist
+module github.com/bitrise-io/go-plist
+
+go 1.15
 
 require (
 	// for cmd/ply
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/kr/pretty v0.1.0 // indirect
+	github.com/stretchr/testify v1.7.0
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
-	gopkg.in/yaml.v2 v2.2.1
+	gopkg.in/yaml.v1 v1.0.0-20140924161607-9f9df34309c0
```

Comment:
why did we switch to yaml.v1?

Reply:
It seems it is referenced in on of the files in the ply package.